### PR TITLE
refactor: token page hooks

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

Future update will remove redux and switch to typescript.

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

### TypeScript/Hooks Update

- [x] Updated files to React Hooks
- [ ] Updated files to TypeScript

### CODE REVIEW

File: src/containers/Token/index.jsx [hunk 1]
```diff
@@ -35,16 +36,21 @@ ERROR_MESSAGES.default = {
...
-    return {
-      accountId: match.params.id,
-      prevId: prevState && prevState.accountId,
+const Token = ({ error }) => {
+  const { currency, id: accountId } = useParams()
+  const { t } = useTranslation()
+  const showError = error
+
+  useEffect(() => {
+    document.title = `${t('xrpl_explorer')} | ${accountId.substr(0, 12)}...`
```
Comment (justinr1234) [inline src/containers/Token/index.jsx:45]: Why not use helmet?

File: src/containers/Token/index.jsx [hunk 2]
```diff
@@ -53,57 +59,32 @@ class Token extends Component {
...
-        {!accountId && (
-          <div style={{ textAlign: 'center', fontSize: '14px' }}>
-            <h2>Enter an account ID in the search box</h2>
-          </div>
-        )}
-      </div>
-    )
-  }
+  return showError ? (
+    renderError(error)
```
Comment (justinr1234) [inline src/containers/Token/index.jsx:63]: Why not refactor to a component? <Error

if(error)  {
  return <Error
}

return <div

**Review by justinr1234 — APPROVED** [review]

File: src/containers/Token/index.jsx (hunk 2 above)
Comment (ckniffen) [reply src/containers/Token/index.jsx:63]: I will create a ticket to do this across the board.  These were existing methods and it is how it is done in many places throughout the app.

File: src/containers/Token/index.jsx (hunk 1 above)
Comment (ckniffen) [reply src/containers/Token/index.jsx:45]: This is how it is done throughout the app.  I will create a ticket to update this to helmet everywhere.

Comment (ckniffen) [reply src/containers/Token/index.jsx:45]: #650

File: src/containers/Token/index.jsx (hunk 2 above)
Comment (ckniffen) [reply src/containers/Token/index.jsx:63]: Created #651

**Pull request merged**